### DISCUSSION TITLE
Enable Issues for the zephyrproject-rtos/example-application repo

### DISCUSSION THREAD

**Discussion by IvanVnucec** [zephyrproject-rtos/zephyr#59983] [General]
I want to improve the https://github.com/zephyrproject-rtos/example-application repo but there is no way to create an Issue. I think this would improve repo quality.

**Comment by nordicjm** [comment]
You can submit pull requests to it?

**Reply by IvanVnucec** [reply]
ok

**Reply by IvanVnucec** [reply]
Well, I can't because what code would I've pulled? This request deals with the GitHub "stuff".
